File: Android/get_Resource_ID/app/src/main/java/com/android/myapplication/MainActivity.java
package com.android.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.media.MediaPlayer;
import android.os.Bundle;
import android.util.Log;
import android.view.View;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    public void buttonTapped(View v) {

        int id = v.getId(); // 해당 View의 id값 반환

        String ourId = v.getResources().getResourceEntryName(id); // 받은 id의 별명 반환

        int resourceId = getResources().getIdentifier(ourId, "raw", "com.android.myapplication"); // 파일명, 파일이 들어있는 패키지(폴더)명, 프로젝트의 패키지명
        // 패키지 안의 raw폴더에서 ourId와 일치하는 파일의 id 반환

        MediaPlayer mplayer = MediaPlayer.create(this, resourceId);

        mplayer.start();

    }

}

File: Android_Kotlin/QuizApp/app/src/main/java/com/android/quizapp/MainActivity.kt
package com.android.quizapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.widget.Toast
import com.android.quizapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_FULLSCREEN // 상단 상태바 제거(시계 LTE 등)

        binding.btnStart.setOnClickListener{
            if(binding.etName.text.toString().isEmpty()) {
                Toast.makeText(this, "Please Enter your name", Toast.LENGTH_SHORT).show()
            }else{
                Toast.makeText(this, "Welcome, ${binding.etName.text.toString()}!" , Toast.LENGTH_SHORT).show()
                val intent = Intent(this, QuestionActivity::class.java)
                startActivity(intent)
                finish()
            }
        }
    }
}